src/grt/src/fastroute/src/utility.cpp

```diff
@@ -1292,7 +1256,7 @@ void FastRouteCore::StNetOrder()
 
   // Set the 70% (or less) of non critical nets that doesn't have overflow
   // with the lowest priority
-  for (int ord_elID = 0; ord_elID < netCount(); ord_elID++) {
+  for (int ord_elID = 0; ord_elID < dirty_net_ids_.size(); ord_elID++) {
```

Comment:
Why use the dirty_net_ids_ size here instead of iterating over each vector item, like everywhere else?